### PR TITLE
Revive GangaTutorial package #1566

### PR DESCRIPTION
**Pull request by dilawari2008**
Doesn't throw error anymore

### CODE REVIEW

File: ganga/GangaTutorial/__init__.py [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 def loadPlugins( config = {} ):
 
-    import Lib.PrimeFactorizer
-    import Lib.PrimeFactorizerSplitter
-    import Lib.PrimeTableDataset
+    import GangaTutorial.Lib.PrimeFactorizer
+    import GangaTutorial.Lib.PrimeFactorizerSplitter
```
Comment (egede) [inline ganga/GangaTutorial/__init__.py:6]: I think this would better be
```
    import .Lib.PrimeFactorizer
```
and the same for all the other lines.

**Review by egede — CHANGES_REQUESTED** [review]
This is a good start. However, the plugin is still not working. The code below should work and return a valid result.
```
j = Job()
j.application=PrimeFactorizer(number=20)
j.inputdata = PrimeTableDataset(table_id_lower=1, table_id_upper=1)
j.submit()
```
There are (at least) a further two problems. There is an error on submission:
```
Ganga In [9]: j.submit()                                                                                 
INFO     submitting job 178
INFO     job 178 status changed to "submitting"
INFO     Preparing subjobs
ERROR    Exception raised executing '_prepare_sj' in Thread 'None':
Traceback (most recent call last):
  File "/home/egede/programming/ganga/src/ganga/ganga/GangaCore/Core/GangaThread/WorkerThreads/WorkerThreadPool.py", line 124, in __worker_thread
    result = item.command_input.function(*these_args, **item.command_input.kwargs)
  File "/home/egede/programming/ganga/src/ganga/ganga/GangaCore/GPIDev/Lib/Job/Job.py", line 1241, in _prepare_sj
    if app.is_prepared in [None, False]:
AttributeError: 'PrimeFactorizer' object has no attribute 'is_prepared'
```
This problem will just require a minor modification to the schema of the object I believe, like adding a line equivalent to https://github.com/ganga-devs/ganga/blob/e5b5da6f449f6c22cf6b7029896468152e738f5c/ganga/GangaCore/Lib/Executable/Executable.py#L59

A second problem is that the location `http://cern.ch/diane/download/primes` where the prime number table is picked up from is no longer valid. A different location should be found.

File: ganga/GangaTutorial/__init__.py (hunk 1 above)
Comment (dilawari2008) [reply ganga/GangaTutorial/__init__.py:6]: @egede 
the line
import .Lib.PrimeFactorizer
is showing the following error
Screenshot from 2020-02-13 13-50-02

Comment (egede) [reply ganga/GangaTutorial/__init__.py:6]: Sorry, look at https://github.com/ganga-devs/ganga/blob/e5b5da6f449f6c22cf6b7029896468152e738f5c/ganga/GangaDirac/__init__.py#L111 for the syntax.

**Comment by egede** [comment]
Stale. Will be fixed by #1615 instead.

**Pull request closed without merging**